util/efd_sched.py

```diff
@@ -0,0 +1,231 @@
+"""Developer utilities to query the EFD for schedulers, and download them.
+"""
+
+import argparse
+import asyncio
+import lzma
+import os
+import pickle
+import re
+
+import astropy.units as u
+from astropy.time import Time
+from lsst.resources import ResourcePath
+from lsst_efd_client import EfdClient
+from rubin_sim.utils import Site
+
+
+async def query_schedulers_in_window(desired_time, efd="usdf_efd", time_window_seconds=2):
+    """Query the EFD for scheduler URLs within a given time window.
+
+    Parameters
+    ----------
+    desired_time : `str`
+        The central time to query the event service in ISO format
+        (YYYY-MM-DDTHH:MM:SS).
+    efd : `str`, optional
+        The name of the EFD to connect to (default is 'usdf_efd').
+    time_window_seconds : `float`, optional
+        The size of the time window to search for scheduler URLs, in seconds
+        (default is 2).
```

Comment:
Does it make sense to make "time" and "time_window" be astropy Time and DeltaTimes? Or is it not working so well with the place you want to place this within the dashboard, because the dashboard wants TimeStamps or strings?

Reply:
Done.